itl/command-plugins-manubulon.conf

```diff
@@ -72,6 +72,14 @@ template CheckCommand "snmp-manubulon-command" {
 			value = "$snmp_timeout$"
 			description = "Timeout for SNMP in seconds (Default: 5)"
 		}
+        "-m" = {
+            value = "$snmp_memcached$"
+            description = "Include cached memory in used memory"
+        }
+        "-b" = {
+            value = "$snmp_membuffer$"
+            description = "Exclude buffered memory in used memory"
+        }
```

Comment:
This doesn't fit here in the global template for all plugins. It might work for snmp-memory but break other CheckCommand definitions.